omod-common/src/test/java/org/mockingbird/test/rest/resource/BirdResource_1_9.java

```diff
@@ -60,6 +61,21 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		return null;
 	}
 	
+	@Override
+	public Model getGETModel(Representation representation) {
+		return null;
+	}
+	
```

Comment:
@dkayiwa 
Yeah I see. But isn't that good that we enforce them? So when creating a resource class one knows that he should document them? BTW I'm following a similar approach as it's done in old getRepresentationDescription